src/ontology/ro-edit.owl

```diff
@@ -6843,6 +6844,22 @@ SubClassOf(obo:BFO_0000015 ObjectAllValuesFrom(obo:RO_0002214 obo:BFO_0000015))
 
 SubClassOf(obo:CHEBI_50906 obo:PATO_0000001)
 
+# Class: obo:COB_0001000 (exposure of organism)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "CTD:curators") obo:IAO_0000115 obo:COB_0001000 "An interaction between an exposure stressor and an exposure_receptor.")
+AnnotationAssertion(obo:IAO_0000116 obo:COB_0001000 "Bill Duncan 2022-05-06:
+In the future, this class may become obsoleted if 'exposure event' is accepted as a class in COB. For now, we are using the class from ECTO.
```

Comment:
Instead of adding the COB term annotations in the edit file, need to update the COB import running `sh run.sh make refresh-cob`

Reply:
> Instead of adding the COB term annotations in the edit file, need to update the COB import running `sh run.sh make refresh-cob`

Can you please handle this. I am out the infrastructure loop. It seem you and Nico have taken this over.

Reply:
@anitacaron and @matentzn will you be handling this? It seems when I touch the RO infrastructure I break things.

Reply:
Oh, sorry, I lost track of this one. I'll make the change.

Reply:
Thanks @anitacaron. You're the best!!!

Reply:
done